backend/src/routes/admin.ts

```diff
@@ -99,6 +104,39 @@ const adminRoutes: FastifyPluginAsync = async (server) => {
     }
   })
 
+  server.post('/updateKey', async function (request, reply) {
+    try {
+      const body: any = JSON.parse(request.body as string);
+      if (!body) return reply.code(ReturnCode.FAILURE).send({ error: ErrorMessage.EMPTY_BODY });
+      if (!body.API_KEY)
```

Comment:
Please add additional validation of the API key. Use regular expressions to determine that the key is of a proper format. This will protect against SQLInjection attacks.

Reply:
what validation of the API key are you suggesting?

Reply:
Whether the API-Key is of a given format. You shouldn't allow any arbitrary text as valid parameter.

Reply:
This doesn't look like it's resolved @vignesha22

Reply:
Did the API_KEY format verification upon creating a new record since that where sql injection could occur and on select cmds its not needed right? @IAmKio

Reply:
Just looked at it again and i thought this endpoint would add / set / update a new API key but it just updates the settings.